validator_client/src/beacon_node_fallback.rs

```diff
@@ -473,20 +528,13 @@ impl<T: SlotClock, E: EthSpec> BeaconNodeFallback<T, E> {
     /// First this function will try all nodes with a suitable status. If no candidates are suitable
     /// or all the requests fail, it will try updating the status of all unsuitable nodes and
     /// re-running `func` again.
```

Comment:
> it will try updating the status of all unsuitable nodes and re-running `func` again.

This behaviour no longer exists in this function and this line can be removed I think.

Reply:
Fixed in [9c206d5](https://github.com/sigp/lighthouse/pull/4393/commits/9c206d5d6ed70e9c3e626a39a0c2165bfe3c4aac)